pkg/aggregator/aggregator.go

```diff
@@ -129,8 +129,10 @@ var (
 		[]string{"data_type", "state"}, "Number of metrics/service checks/events flushed")
 	tlmProcessed = telemetry.NewCounter("aggregator", "processed",
 		[]string{"data_type"}, "Amount of metrics/services_checks/events processed by the aggregator")
+	tlmDogstatsdContextsWithOrigin = telemetry.NewCounter("aggregator", "dogstatsd_contexts_with_origin",
+		[]string{"origin"}, "Count the contexts processed by the time samplers, per origin")
 	tlmDogstatsdContexts = telemetry.NewGauge("aggregator", "dogstatsd_contexts",
-		nil, "Count the number of dogstatsd contexts in the aggregator")
+		nil, "How many contexts are processed in the time samplers")
```

Comment:
```suggestion
		nil, "Number of dogstatsd contexts currently tracked by all time samplers")
```

(And this is wrong when pipelineCount > 1, because they would overwrite each other's values [here](https://github.com/datadog/datadog-agent/blob/146b2744a6aa235beba476ae7bec55a547ebafa8/pkg/aggregator/time_sampler.go#L223-L224))